Core/clim-core/builtin-commands.lisp

```diff
@@ -351,11 +351,11 @@
     (if (or subform-read auto-activate)
         (values object ptype)
         (loop
-           for c = (read-char stream)
+           for c = (read-gesture :stream stream)
```

Comment:
Maybe change `c` to `gesture` or similar since that name was probably chosen based on the (incorrect) assumption that the code would operate on characters.

Reply:
fixed